profile_xf05id1/startup/15-optics.py

```diff
@@ -2,9 +2,24 @@
 from ophyd import Device
 from ophyd import Component as Cpt
 
+class SRXSlitsWB(Device):
+    #THIS IS BROKEN AT EPICS :LEVEL
```

Comment:
This might have already been fixed by at the epics level, not something we got to yesterday.